spec.md

```diff
@@ -103,6 +104,27 @@ This header allows brokers to reject requests from marketplaces for versions the
 The marketplace MUST authenticate with the service broker using HTTP
 basic authentication (the `Authorization:` header) on every request. The broker is responsible for validating the username and password and returning a `401 Unauthorized` message if credentials are invalid. It is RECOMMENDED that brokers support secure communication from platform marketplaces over TLS.
 
+## Originating Identity
+
+Often a service broker will need to know the identity of the user that
+initiated the request from the platform. For example, this might be needed
+for auditing or authorization purposes. In order to facilitate this, the
+platform will need to provide this identification information to the broker
+on each request. If the platform wishes to provide this information then
+it MUST include an additional HTTP header on all requests it sends to the
+broker, except the request to retrieve the catalog since that is not
+associated with a platform's user. The format of the header MUST be:
+
+```
+X-Broker-API-Originating-Identity: value[;charset=...]
+```
+Where:
+- its REQUIRED `value` is a non-empty string.
+- it MAY include a `charset` parameter specifying the character encoding of
+  the `value`. If not specified, the default value MUST be `us-ascii`.
```

Comment:
Also, do we want to discuss that the `value` may be any scheme?  For example, in Kubernetes, we'll probably want to send the base64 encoded userinfo.  See https://github.com/kubernetes-incubator/service-catalog/pull/939